airflow/providers/databricks/operators/databricks.py

```diff
@@ -671,6 +671,9 @@ def execute(self, context: Context):
         else:
             _handle_databricks_operator_execution(self, hook, self.log, context)
 
+    def execute_complete(self, context: dict | None, event: dict):
+        _handle_deferrable_databricks_operator_completion(event, self.log)
```

Comment:
```suggestion
         if self.deferrable:
               _handle_deferrable_databricks_operator_completion(event, self.log)
```

Reply:
I'm not sure whether we should add `if self.deferrable` as this is only called when `self.deferrable is Ture` and this pattern does not exist elsewhere

Reply:
we don't have different operators for deferrable and sync versions as well which is not what we followed till now

Reply:
Yes, but other operators (e.g., [BatchOperator](https://github.com/apache/airflow/blob/main/airflow/providers/amazon/aws/operators/batch.py#L239)) with both modes doesn't seem to apply this pattern either